packages/openai/src/openai-chat-language-model.test.ts

```diff
@@ -224,6 +332,24 @@ describe('doGenerate', () => {
     expect(usage).toStrictEqual({ inputTokens: 20, outputTokens: undefined });
   });
 
+  it('should extract logprobs', async () => {
```

Comment:
does this work for streaming? is streaming tested?

Reply:
done